solr/modules/ltr/src/java/org/apache/solr/ltr/model/MultipleAdditiveTreesModel.java

```diff
@@ -298,7 +307,9 @@ private static float scoreNode(float[] featureVector, RegressionTreeNode regress
         return 0f;
       }
 
-      if (featureVector[regressionTreeNode.featureIndex] <= regressionTreeNode.threshold) {
+      if ((featureVector[regressionTreeNode.featureIndex] <= regressionTreeNode.threshold) ||
```

Comment:
'left' or 'right are the only values here, so I would change the variable to use enum instead.
The you can isolate the condition here, to check with the enum.
It's always discouraged to use checks with strings( always prefer constants when possible)